routes/feeds.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+use Daalder\Feeds\Models\Feed\FeedsPermission;
+use Illuminate\Support\Facades\Route;
+use Daalder\Feeds\Http\Controllers\FeedsController;
+use Illuminate\Support\Str;
+
+$middleware = hook('authenticated-middleware',
```

Comment:
Please move the $middleware and $group_attributes to the route server provider, you can load this file within a group, Daalder also does that for web and api routes.